test-jest/chart/LineChart.spec.tsx

```diff
@@ -61,6 +65,11 @@ describe('<LineChart />', () => {
     const curves = container.querySelectorAll('.recharts-line .recharts-line-curve');
     expect(curves).toHaveLength(1);
     const path = curves[0].getAttribute('d');
+
+    if (!path) {
+      throw new Error('Path is null');
+    }
+
     expect(path.indexOf('C')).not.toEqual(-1);
```

Comment:
Instead of throwing an error you could simply use conditional invocation. 

```suggestion
    expect(path?.indexOf('C')).not.toEqual(-1);
```